client/js/components/StageTwo/StageTwo.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from "react";
+import {
+  ScrollView,
+  View,
+  Text,
+  Image,
+  TouchableOpacity,
+  Dimensions
+} from "react-native";
+import * as Progress from "react-native-progress";
+import styles from "./styles";
+import theme from "../../config/styles";
+export default class StageTwo extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      clickedT: false,
+      clickedF: false
```

Comment:
You just need one state to see if the submitted answer was true or false. Please make the changes.